using-kite-properties.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: page
+title: Using Kite Properties
+---
+
+Kite gives you the option of changing settings of existing system properties to enhance performance. You also have the option of creating custom properties for additional control over how Kite reads or writes your data.
+
+## Setting Kite Properties
+
+There are several dataset properties that affect how files are written. These settings are especially relevant for Parquet because it buffers records in memory. 
+
+You can set properties on your datasets with [`DatasetDescriptor.Builder.property`][dataset-descriptor-builder] when using the API, or with the `--set` option from the command line.
+
+### kite.write.cache-size
```

Comment:
Changed this in the header and the next sentence. Filed CDK-901 to be sure this is fixed in the update -h output.